2018/10/16/pom.xml

```diff
@@ -17,6 +17,10 @@
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-compiler-plugin</artifactId>
        <version>2.3.2</version>
+       <configuration>
+	       <source>1.8</source>
```

Comment:
Modern versions of Java and/or Maven can't compile Java 1.5 source any more, and the compiler plugin defaults to 1.5 (not sure why it worked for you, maybe you are on an older java version). Java 8 is a reasonable default for Java today.